crates/paimon/src/spec/manifest_list.rs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use super::manifest_file_meta::ManifestFileMeta;
+
+/// This file includes several [`ManifestFileMeta`], representing all data of the whole table at the corresponding snapshot.
+pub struct ManifestList {}
+
+impl ManifestList {
+    /// Write several [`ManifestFileMeta`]s into a manifest list.
+    ///
+    /// NOTE: This method is atomic.
+    pub fn write(&mut self, _metas: Vec<ManifestFileMeta>) -> &str {
+        todo!()
+    }
+}
+
+impl ManifestList {
+    /// Create ManifestList.
+    pub fn create() -> ManifestList {
```

Comment:
Why create another `impl ManifestList` for this function?

Reply:
I usually put the associated method in another impl

Reply:
Hi, it's best to group all functions together for easier finding. How about using `new()` instead of `create()`?

Reply:
@Xuanwo done